osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/objectconfig/Attribute.java

```diff
@@ -14,5 +15,19 @@ public class Attribute {
   private DlmsDataType datatype;
   private ValueType valuetype;
   private String value;
+  private List<TypeBasedValue> values;
   private AccessType access;
+
+  public Attribute() {}
+
+  public Attribute(final Attribute attribute) {
```

Comment:
Vind dit een bijzondere constructor...... Is het niet beter om hier een methode van te maken?

Reply:
Changed to a copy() function